contracts/pco-license/libraries/LibCFAPenaltyBid.sol

```diff
@@ -80,18 +80,31 @@ library LibCFAPenaltyBid {
             .diamondStorage();
         LibCFABasePCO.DiamondCFAStorage storage cs = LibCFABasePCO.cfaStorage();
         LibCFABasePCO.Bid storage _currentBid = LibCFABasePCO._currentBid();
-        Bid storage _pendingBid = pendingBid();
+        LibCFABasePCO.Bid memory oldCurrentBid = _currentBid;
+        Bid memory _pendingBid = pendingBid();
         ISuperToken paymentToken = ds.paramsStore.getPaymentToken();
 
```

Comment:
I don't see any checks to see if a flow with the required flowRate can be created for the user

Reply:
This should be taken care of when it calls `_editBid`